en/publish/publish-wechatgame.md

```diff
@@ -5,111 +5,138 @@ The runtime environment of the WeChat Mini Game is an extension of the WeChat Mi
+This brings up two key issues, one is the downloading, caching and versioning of remote resources, and the other is the loading speed of the start scene. Let's look at the logic of this part below, note that this section is updated in v2.4, so if you are using a version before v2.4, please take care to switch to the corresponding documentation branch.
 
-In a mini game environment, the process of downloading resources from the engine is as follows:
+### Downloading, Caching and Versioning of Remote Resources
 
-1. Check if the resource is in the mini game pack
-2. Check if the resource is in local cache storage if it's not in game pack
-3. Check if the resource is in the temporary directory if it's not in cache
-4. Download from a remote server if they do not exist in mini game pack
-5. After downloading to temporary directory, use it directly
-6. Save it to the game application cache slowly in backstage for re-access.
-7. Local cache storage has space limitation, if total space of cache exceeds the limit, the LRU algorithm is used to delete older resources.
+As of v2.4.0, all resources are managed in an [Asset Bundle](../asset-manager/bundle.md). For downloading, caching, and versioning of remote resources, Creator has already done that for you. Let's look at the process of the engine downloading resources in the mini game environment:
 
-It should be noted that once the cache space is full, all the resources that need to be downloaded cannot be saved, only the temporary files for save download resources can be used, and WeChat will automatically clean up all temporary files after the mini game is exited. So the next time you run the mini game again, those resources are downloaded again and the process keeps looping.  
-In addition, the problem of file saving failure due to cache space exceeding the limit does not occur on the WeChat DevTools, because the WeChat DevTools does not limit the cache size, so testing the cache needs to be done in a real WeChat environment.
+1. Check if the resource is in the mini game package
+2. If not, check if the resource is in the local cache
+3. If not, check if the resource is in the temporary file
+4. If not, download the resource from the remote server
+5. After downloading the resource to the temporary file, use it directly
+6. The background slowly saves the resource from the temporary file to the mini game application cache for reuse
```

Comment:
Slowly save the resource from the temporary file to the local cache for reuse in the background